package/MDAnalysis/lib/formats/include/readdcd.h

```diff
@@ -136,7 +136,7 @@ static int write_dcdheader(fio_fd fd, const char *remarks, int natoms,
  * Output: 0 on success, negative error code on failure.
  * Side effects: coordinates are written to the dcd file.
  */
-static int write_dcdstep(fio_fd fd, int curstep, int curframe, 
```

Comment:
You think curframe should remain the third argument? Check the order of the descriptions in the comment block above. Also, check a more recent version of the dcdplugin source code: http://www.ks.uiuc.edu/Research/vmd/plugins/doxygen/dcdplugin_8c-source.html

```C
00711 static int write_dcdstep(fio_fd fd, int curframe, int curstep, int N, 
00712                   const float *X, const float *Y, const float *Z, 
00713                   const double *unitcell, int charmm) {
```
It seems the original author has also changed the order of the arguments, so I'd hardly characterize the swap as having no reason. I'm also curious why we're using such an old version of this C code? How did it end up here? It seems to be at least 12 years older than versions of similar code I can find online.